### PR TITLE
Create s3 storage

### PR DESCRIPTION
**Pull request by bilalesi**
Currently projects can not create files and save it in local storage (due deprecation in favor of S3),
this PR, will allow the project to instantiate an s3 storage in nexus.

**NOTE**: I delete the config for docker-compose-p (created for stripe integration long time ago, since it's not relevant anymore, the current config will handle all cases, and stripe features will be removed soon from this service).

### CODE REVIEW

**Comment by Dinika** [comment]
Thanks Bilal. I'll start having a look asap.

File: dev-init.sh [hunk 1]
```diff
@@ -10,6 +10,8 @@ CLIENT_SECRET="obp-secret"
 
 
 # Start containers
+chmod +x ./env-prep/init/init-aws.sh
+ls -l ./env-prep/init/init-aws.sh
```
Comment (Dinika) [inline dev-init.sh:14]: I don't think these two lines are needed. The file's executable nature is persisted on git.

**Review by Dinika — APPROVED** [review]
Looks good to me. Left one comment.

File: dev-init.sh (hunk 1 above)
Comment (Dinika) [reply dev-init.sh:14]: I also tested this right now and it seems to work for me without these lines.

**Review by pgetta — APPROVED** [review]
Thank you, looks good.

File: dev-init.sh (hunk 1 above)
Comment (bilalesi) [reply dev-init.sh:14]: well I had to do it, and also in the docs they mention that, so probably depends on `os`, or you start `make` command with `sudo` privileges  ?

**Pull request merged**